include/core.h

```diff
@@ -19,14 +19,25 @@ namespace globals {
     extern fs::perms file_permissions;
 }
 #endif
-#define UNIT_TEST
+
+#define PLUGIN_ERRORS (-1)
+#define INFO 0
+#define VERBOSE_1 1
+#define VERBOSE_2 2
+
+#define LOGGING_LEVEL VERBOSE_1
 #ifndef NDEBUG
     #ifndef UNIT_TEST
-     #define DEBUG_LOG(msg) if(globals::context) OrthancPluginLogWarning(globals::context, msg);
+     #define DEBUG_LOG(L,msg) if(globals::context && L <= LOGGING_LEVEL) switch(L){ \
```

Comment:
I do wonder if this should be moved to a new file `debug.h` but it's honestly not that big, so I figure why bother.

Reply:
I agree if it gets bigger maybe we move it to its own file but for now, this should be fine